pgctl/__init__.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+__version__ = '0.2.1'
```

Comment:
# NOTE: this file is imported by setup.py, so cannot have any dependencies or side-effect.